index.html

```diff
@@ -5,25 +5,143 @@
+          <li class="list-item2b">Presentation</li>
+        </ul>
+      </nav>
+    </div>
   </header>
+  <h1 class="not-appear">FICA Soccer Tournament</h1>
+  <section class="section1-wrapper">
+    <h2 class="not-appear">FICA Soccer Tournament</h2>
+    <div class="section1">
+      <img id="title" src="img/title.png" width=324 height=110 alt="FICA 2022 Tournament">
+      <p class="text-box"> A fun festival where teams from 10 countries who believe in the value
+        of friendship and sports gather and create positive change, which will be held in Buenos Aires in September.
+      </p>
+      <p class="schedule"></p><div class="sched-text">2022.09.14 (Wed) ~ 18 (Sun)</div>
+        <div class="big">@ River Plate Stadium of Buenos Aires, Belgrano, etc.</div><p></p>
+    </div>
+  </section>
 
-  <section>
+  <section class="section2-wrapper">
+    <div class="section2">
+      <h2 class="myh1">Main Program</h2>
+      <hr class="hr2">
+      <div class="programs-wrapper">
+        <div class="program-block">
+          <img src="img/program_icon_01.png" width=64 height=58 alt="Presentation">
+          <div class="subtitle">Presentation</div>
+          <div class="description">The 10 teams will be presented together with their background and records, and key players.</div>
+        </div>
+        <div class="program-block">
+          <img src="img/program_icon_02.png" width=64 height=58 alt="The Games">
+          <div class="subtitle">The Games</div>
+          <div class="description">The 16 games divided into four days, with a minimum of four games per team will be played.</div>
+        </div>
+        <div class="program-block">
+          <img src="img/program_icon_03.png" width=64 height=58 alt="Press conference">
+          <div class="subtitle">Conference</div>
+          <div class="description">After each game two team members, one of them being the coach, will answer questions to the press.</div>
+        </div>
+        <div class="program-block">
+          <img src="img/program_icon_04.png" width=64 height=58 alt="Dissertation">
+          <div class="subtitle">Dissertation</div>
+          <div class="description">The closing day a dissertation by Marcelo Bielsa will precede the tournament game.</div>
+        </div>
+        <div class="program-block">
+          <img src="img/program_icon_05.png" width=64 height=58 alt="Awards">
+          <div class="subtitle">Awards</div>
+          <div class="description">After the closing game, the award ceremony will take place with a small convite for the participants and guests.</div>
+        </div>
+      </div>
+      <div class="texttoright">View Full Program ></div>
+    </div>
+  </section>
 
+  <section class="section3-wrapper">
+    <div class="section3">
+      <h2 class="myh1b">main teams</h2>
+      <hr class="hr2">
+      <div class="teams-wrapper">
+      </div>
+      <div class="texttoright2">See More Teams ></div>
+    </div>
```

Comment:
- [ ] Your Teams section is looking great, good job! Let's make it even better by placing the description to the right of the image instead of below the image:

Template:
![Captura de pantalla 2022-04-14 a la(s) 9 52 34 p  m](https://user-images.githubusercontent.com/83717285/163511286-6262096a-2344-469d-ab02-fefb514ab0bb.png)

Your design:
![Captura de pantalla 2022-04-14 a la(s) 9 52 18 p  m](https://user-images.githubusercontent.com/83717285/163511314-b0977eb4-372b-40c1-90f4-34d886a87e4d.png)